packages/synchro-charts/src/components/sc-widget-grid/sc-grid-tooltip.tsx

```diff
@@ -21,8 +21,6 @@ export class ScGridTooltip {
   @Prop() propertyPoint?: DataPoint;
   @Prop() alarmPoint?: DataPoint;
   @Prop() breachedThreshold?: Threshold;
-  @Prop() unit?: string;
-  @Prop() value?: number | string;
```

Comment:
The tooltip no longer accepts unit and value? This doesn't have impact on other components?